### ISSUE TITLE
Sprint 1 - Self Grade

### ISSUE THREAD

**Issue by avikaprasad22** [avikaprasad22/avika_2025#3]
# Grading
All self and by peer grading must be complete prior to Teacher review date(s).  Show a direct link to the tangible(s) for each line item. 

## Sprint1 Key Objectives
During the sprint you need to complete all of these key objectives and find and complete supporting hacks that show level of command you are declaring.

Rank to Ratio, rank is an integer
0 = .00 work, but faking understanding/accomplishment
1 = .55 incomplete assignment, big gaps
2 = .75 meeting some objectives, multiple little gaps
3 = .85 meeting most objectives, one little gap
4 = .90 meeting all objectives
4+Mastered-Y = .95 exceeding all objectives, requires links and  office hours verification

| Skill                                  | Mastered (Y/N) | Rank (1-4) | Ratio | Notes                                                   |
|----------------------------------------|----------------|------------|-------|---------------------------------------------------------|
| Laptop Verification or Cloud Workspace | [Y]          | 4          | 0.0   | verification by Linux commands notebook                 |
| VSCode                                 | [Y]          | 4          | 0.0   | live change code, commit, make, run localhost, test, debug |
| Show VSCode GitLens                    | [Y]          | 4          | 0.0   | verify commits, show change, comments, reset to commit, run on old commit |
| GitHub student_2025 repo                      | [Y]          | 4          | 0.0   | template create, push changes, verify id, review analytics             |
| GitHub Pages Actions                   | [Y]          | 4          | 0.0   | review runs and verify runtime, receive demo            |
| Hacks Tools & Equipment Play           | [Y]          | 3          | 0.0   |     [Hacks](https://nighthawkcoders.github.io/portfolio_2025/jupyter/notebook/python#hacks)                                                   | Already good, emoji works, maybe could attempt the other examples provided.
| Hacks GitHub Pages Playground          | [Y]          | 3          | 0.0   |    [Hacks]  (https://nighthawkcoders.github.io/portfolio_2025/github/pages/hacks)                                                   | Make the java cell execute a valid output.
| Hacks SASS Basics                      | [Y]          | 4          | 0.0   |    [Hacks](https://nighthawkcoders.github.io/portfolio_2025/sass_basics/play)                                                     |
| Hacks Frontend                 | [Y]          | 4          | 0.0   |    [Hacks](https://nighthawkcoders.github.io/portfolio_2025/frontend/basics/playground)                                                 |
| Pair Programming                       | [Y]          | 4          | 0.0   | tangibles that show coop work                           | have a changing background on the about page that matches my friend's.
| **Total**                              |        38        |            | 3.8   |                                                         |
| **Average**                            |                |            | 0.0   |                                                         |

## Learning Habits Table
Review criteria in habits.  In this review, pairs should rate each other base on their experience with the person on the line item.

Rank to Ratio, rank is an integer  
1 = .55
2 = .75
3 = .85
4 = .90
4+Mastered-Y = .95

| Skill          | Mastered (Y/N) | Rank (1-4)   | Ratio | Notes |
|----------------|----------------|--------------|-------|-------|
| Attendance     | [Y]            | 4            | 0.0   |       | I always attended class.
| Work Habits    | [Y]            | 3.5            | 0.0   |       | I'm almost always on-task.
| Behavior       | [Y]            | 4            | 0.0   |       | I never behaved poorly.
| Timeliness     | [Y]            | 3.5            | 0.0   |       | I did most hacks on time, some were postponed as I needed help.
| Tech Sense     | [Y]            | 4            | 0.0   |       | I understand the changes I have made to my website pretty well.
| Tech Talk      | [Y]            | 3            | 0.0   |       | It was good, but there's always room for improvement.
| Tech Growth    | [Y]            | 4            | 0.0   |       | I learned a lot!
| Advocacy       | [Y]            | 4            | 0.0   |       | I helped my peers when they needed help resolving some issues.
| Comm & Collab  | [Y]            | 4            | 0.0   |       |  I collaborated with my friends to understand the hacks.
| Integrity      | [Y]            | 4            | 0.0   |       | My website is pretty original, focused towards me.
| Organized      | [Y]            | 4            | 0.0   |       | My website is very organized.
| **Total**      |              |      42        | 0.0   |       |
| **Average**    |                |       3.81       | 0.0   |       |
